### PR TITLE
[two-fer] Update instructions.md

### PR DESCRIPTION
**Pull request by fgm**
The test sends multiple lines, but unlike what would be expected from the current description, it only expects the last one to be used. Added a line to explain that.

### CODE REVIEW

File: exercises/practice/two-fer/.docs/instructions.md [hunk 1]
```diff
@@ -24,3 +24,5 @@ Here are some examples:
 |Bob     |One for Bob, one for me.
 |        |One for you, one for me.
 |Zaphod  |One for Zaphod, one for me.
+
```
Comment (glennj) [inline exercises/practice/two-fer/.docs/instructions.md:27]: I wish GFM tables were a bit more flexible. What if we add a row (or replace the Zaphod row) with something like
```none
|(multiple lines) Carl \n Donna \n Eric|One for Eric, one for me.|
```

**Comment by glennj** [comment]
@IsaacG you had the same kind of question about this test earlier. Do you want to chime in?

**Comment by glennj** [comment]
A brain cramp: the instructions.md file is shared between tracks, so it should not be modified.

Any AWK specific instructions need to be added to `instructions.append.md`

**Comment by IsaacG** [comment]
If we don't want students to have to figure out the specific behavior, it might make more sense just to drop that test.

If we do want to bother having this input handled, we do take a TDD approach for most other exercises.

**Comment by glennj** [comment]
Or, the test can have a line of output for each line of input.
```sh
@test "multiple names given" {
  [[ $BATS_RUN_SKIPPED == "true" ]] || skip
  run gawk -f two-fer.awk two.txt
  assert_success
  assert_line --index 0 -- "One for John Smith, one for me."
  assert_line --index 1 -- "One for Mary Ann, one for me."
}
```
That would be less surprising.

**Comment by glennj** [comment]
@fgm would you like to change the approach to this PR and edit the test file?

**Comment by glennj** [comment]
@IsaacG should we remove the multiple-input test, or have the test expect a line of output for each line of input?

**Comment by IsaacG** [comment]
I'm in favor of just dropping it.

**Pull request closed without merging**